docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -256,6 +257,28 @@ test('My test', async t => {
 });
 ```
 
+### Filter Predicate
+
+The `filterFn` function used in `filter`, `find`, `parent`, `child` and `sibling` methods is a predicate that is executed on the client side.
+
+This predicate takes the following parameters.
+
+Property | Description
+------ | -----
+`node`  | The current DOM node.
+`idx` | Index of the current node among other nodes returned by the preceding selector.
```

Comment:
It may be not the preceding selector result (it's preceding selector result only in case of `filter` function), in other cases it's a matching set, e.g. 
```js
Selector('.test').child((el, idx) => ...)
```
In that case `idx` is the index of child, not the `.test` which is preceding selector